src/NearOrg/Notifications/Notification.jsx

```diff
@@ -322,5 +357,35 @@ return (
         {actionable && <Button href={postUrl}>View</Button>}
       </Right>
     </Content>
+    <Widget
+          src="${REPL_ACCOUNT}/widget/DIG.DropdownMenu"
+          props={{
+            trigger: <i className="ph-bold ph-dots-three" />,
+            items: buildMenu(),
+          }}
+        />
+        {isModalOpen && (
+        <div className="modal" style={{ display: 'block', backgroundColor: 'rgba(0, 0, 0, 0.6)' }}>
```

Comment:
For the modal we have a [DIG.Dialog](https://near.org/near/widget/ComponentDetailsPage?src=near/widget/DIG.Dialog&tab=about) component

Reply:
This has been also addressed in the latest commit. Thank you for the review.